model/src/main/scala/aqua/model/transform/ResolveFunc.scala

```diff
@@ -12,7 +12,8 @@ case class ResolveFunc(
   callback: (String, Call) => FuncOp,
   respFuncName: String,
   wrapCallableName: String = "funcAround",
-  arrowCallbackPrefix: String = "init_peer_callable_"
+  arrowCallbackPrefix: String = "init_peer_callable_",
+  returnVar: String = "-return-"
```

Comment:
Is there any reason to keep this? It should always be resolved & replaced internally, never goes to generated code